app/src/main/java/org/schabi/newpipe/player/event/PlayerGestureListener.java

```diff
@@ -33,61 +33,59 @@
 public class PlayerGestureListener
         extends BasePlayerGestureListener
         implements View.OnTouchListener {
-    private static final String TAG = ".PlayerGestureListener";
-    private static final boolean DEBUG = BasePlayer.DEBUG;
+    private static final String TAG = PlayerGestureListener.class.getSimpleName();
```

Comment:
Is this done like this in other places too?

Reply:
Yes, this is the way to do it